doc/installation-ce.rst

```diff
@@ -39,6 +44,16 @@ You will also potentially needs the gitdb module ::
 For npm and webpack, you need a stable version of npm and webpack version 3.11. On Debian
 you can do ::
 
+Add nodesource repository::
```

Comment:
On which distribution is it necessary ?

Reply:
I don't really know, installing npm through apt seems enough for me